.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
+  // Verwendet IntelliSense zum Ermitteln möglicher Attribute.
+  // Zeigen Sie auf vorhandene Attribute, um die zugehörigen Beschreibungen anzuzeigen.
+  // Weitere Informationen finden Sie unter https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
??

Reply:
Sorry putting it in .gitignore. Good catch!